src/frontend/src/pages/FinancePage/ReimbursementRequestPage/ReimbursementRequestPage.tsx

```diff
@@ -0,0 +1,176 @@
+    return (
+      <Box>
+        <Typography variant="h5">Basic Information</Typography>
+        <Grid container spacing={2} sx={{ ml: 2, mt: 2 }}>
+          <Grid item xs={12}>
+            <DetailDisplay label="Recipient" content={fullNamePipe(reimbursementRequest.recipient)} />
+          </Grid>
+          <Grid item xs={12}>
+            <DetailDisplay label="Purchased From" content={reimbursementRequest.vendor.name} />
+          </Grid>
+          <Grid item xs={12}>
+            <DetailDisplay label="Refund Source" content={`${reimbursementRequest.account}`} />
+          </Grid>
+          <Grid item xs={12}>
+            <DetailDisplay label="Date of Expense" content={`${datePipe(new Date(reimbursementRequest.dateOfExpense))}`} />
+          </Grid>
+          <Grid item xs={12}>
+            <DetailDisplay label="Expense Type" content={`${reimbursementRequest.expenseType.name}`} />
+          </Grid>
+        </Grid>
+      </Box>
+    );
+  };
+
+  const ReimbursementProductsView = () => {
+    const uniqueWbsElementsWithProducts = new Map<String, ReimbursementProduct[]>();
+    reimbursementRequest.reimbursementProducts.forEach((product) => {
+      const wbs = wbsPipe(product.wbsNum);
+      if (uniqueWbsElementsWithProducts.has(wbs)) {
+        const products = uniqueWbsElementsWithProducts.get(wbs);
+        products?.push(product);
+      } else {
+        uniqueWbsElementsWithProducts.set(wbs, [product]);
+      }
+    });
+
+    const keys = [];
+    for (const key of uniqueWbsElementsWithProducts.keys()) {
+      keys.push(key);
+    }
+
+    return (
+      <div ref={ref}>
+        <Typography variant="h5">Products</Typography>
+        <Table sx={{ mx: 2 }}>
+          <TableHead>
+            <TableRow>
+              <TableCell>WBS Element</TableCell>
+              <TableCell>Products</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {keys.map((key) => {
+              return (
+                <TableRow>
+                  <TableCell>{key}</TableCell>
+                  <TableCell>
+                    {uniqueWbsElementsWithProducts.get(key)?.map((product) => {
+                      return <Chip label={`${product.name} $${product.cost}`} />;
+                    })}
+                  </TableCell>
+                </TableRow>
+              );
+            })}
+          </TableBody>
+        </Table>
+      </div>
+    );
+  };
+
+  const ReceiptsView = () => {
+    return (
+      <Box sx={{ maxHeight: `${height!}px`, overflow: 'auto' }}>
+        <Typography variant="h5">Receipts</Typography>
+        {reimbursementRequest.receiptPictures.map((receipt) => {
+          return <iframe src={`https://drive.google.com/file/d/${receipt.googleFileId}/preview`} title="ollie"></iframe>;
+        })}
+      </Box>
+    );
+  };
+
+  return (
+    <Box>
+      <PageTitle
+        title={`${fullNamePipe(reimbursementRequest.recipient)}\`s Reimbursement Request`}
```

Comment:
This is different from the mocks, we need to put the sabo # somewhere if you're gonna do it this way, also users can have multiple reimbursement requests

Reply:
ik but not all of them have a sabo #